.github/ISSUE_TEMPLATE/navigator_bug_report.yml

```diff
@@ -0,0 +1,87 @@
+name: Navigator Bug Report
+description: Create Navigator bug report to help us improve
+labels: ['type-bug','pkg-navigator']
```

Comment:
Should we add a type label right after an issue is created? As I remember type labels initiate the creation of corresponding Jira tickets in the anaconda-issues project. So we may have a lot of "invalid" issues that are already moved to Jira.

We may leave this but change the sync logic to rely on another label or just leave the issue without type, so issue will get a type label from the triage engineer after review/high-level investigation.

Reply:
That's true, I suggest we add a new label like we do with the conda issues where only something with `anaconda-sync` label will sync.

I can update our Anaconda internal tool that syncs them.

Reply:
sync label works for me. I'll add it to my triaging docs